src/components/CircleCiBuildDetails/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import './styles.css';
+
+class CircleCiBuildDetails extends Component {
+  constructor(props) {
+    super();
+  };
+
+  render() {
+    const { buildNo, projectName, branchName, status, commitMsg } = this.props.commitDetails;
+    return (
+      status ?
+      <div className='circle-ci-build-details'>
+        <h2>Circle Ci</h2>
+        <div>
+          <label>Build no #{buildNo}</label>
+        </div>
+        <div className='details-data'>
+          Project name: {projectName}
+        </div>
+        <div className='details-data'>
+          Branch name: {branchName}
+        </div>
+        <div className='details-data'>
+          Status: <label className={'status-' + status}>{status.charAt(0).toUpperCase() + status.slice(1)}</label>
```

Comment:
Either move this out as capitalize utility function, or use loads utility :). + it'd be the most proper capitalization if the rest was lower-cased